ouroboros-consensus-shelley-test/src/Test/Consensus/Shelley/Examples.hs

```diff
@@ -19,15 +19,31 @@ module Test.Consensus.Shelley.Examples (
   , examplesShelley
   ) where
 
+import qualified Data.Map as Map
+import           Data.Maybe (fromJust)
```

Comment:
Please use `\case` etc or a local function definition instead of `fromJust`; the errors from that failing are entirely void of information.